app/src/main/java/at/ac/fhcampuswien/budget_fox/widgets/SimpleDropdownField.kt

```diff
@@ -0,0 +1,58 @@
+package at.ac.fhcampuswien.budget_fox.widgets
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.BasicText
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun SimpleDropdownField(
+    placeholder: String,
+    items: List<String>,
+    onItemSelected: (String) -> Unit
+) {
+    var expanded by remember { mutableStateOf(false) }
+    var selectedItem by remember { mutableStateOf<String?>(null) }
+
+    Column {
+        Box(modifier = Modifier.fillMaxWidth().padding(top = 10.dp)) {
+            BasicText(
+                text = selectedItem ?: placeholder,
+                style = MaterialTheme.typography.bodyLarge.copy(color = Color.Black),
```

Comment:
```suggestion
                style = MaterialTheme.typography.bodyLarge.copy(color = MaterialTheme.colorScheme.inverseSurface),
```

Make it work in dark mode.
Before:
<img width="200" alt="Bildschirmfoto 2024-06-26 um 13 33 26" src="https://github.com/EP1210/Budget_Fox/assets/58122000/b8f2e64b-4d2d-4aaf-a87b-cf9b743f1ec0">
After:
<img width="176" alt="Bildschirmfoto 2024-06-26 um 13 38 48" src="https://github.com/EP1210/Budget_Fox/assets/58122000/e20dec1b-c605-4e5b-84c0-de1a2111f023">